Makefile

```diff
@@ -74,14 +74,14 @@ $(BINARIES): bin/%:
 mod-tidy:
 	$(Q)for mod in $$(find . -name go.mod); do \
 	    echo "Tidying $$mod..."; ( \
-	        cd $$(dirname $$mod) && go mod tidy \
+	        cd $$(dirname $$mod) && go mod tidy -go=$$(grep -E "^go\s+\d\.\d+$$" go.mod | sed 's/^go //') \
```

Comment:
`grep -E "^go\s+\d\.\d+$$" go.mod` doesn't output anything in my env:
```
$ grep -E "^go\s+\d\.\d+$" go.mod || echo nope
nope
```

I think `grep` doesn't understand `\d `pattern, does it?

Reply:
```
container-device-interface git:(producer-api) ✗ grep -E "^go\s+\d\.\d+$" go.mod | sed 's/^go //'
1.20
```

Reply:
```➜  container-device-interface git:(fix-go-mod-tidy) ✗ docker run --rm -ti -v $(pwd):/work -w /work ubuntu grep -E "^go\s+[1-9]\.[0-9]+$" go.mod
go 1.20
➜  container-device-interface git:(fix-go-mod-tidy) ✗ grep -E "^go\s+[1-9]\.[0-9]+$" go.mod
go 1.20```